extend.sh

```diff
@@ -331,17 +331,19 @@ function resize_fs(){
 }
 
 function shrink_partition() {
-    new_end_partition_size_in_bytes=$(calculate_new_end_partition_size_in_bytes "$1")
-    ret=$(echo Yes | /usr/sbin/parted "$DEVICE_NAME" ---pretend-input-tty unit B resizepart "$1" "$new_end_partition_size_in_bytes" 2>&1 )
+    local partition_number=$1
+    new_end_partition_size_in_bytes=$(calculate_new_end_partition_size_in_bytes "$partition_number")
+    ret=$(echo Yes | /usr/sbin/parted "$DEVICE_NAME" ---pretend-input-tty unit B resizepart "$partition_number" "$new_end_partition_size_in_bytes" 2>&1 )
     status=$?
     if [[ $status -ne 0 ]]; then
-        echo "Failed to resize device $DEVICE_NAME$1 to size: $ret"
+        echo "Failed to resize device $DEVICE_NAME$partition_number to size: $ret"
         exit 1
     fi
 }
 
-function shrink_target_partition(){
-    device_type=$(get_device_type "$DEVICE_NAME$SUCCESSIVE_PARTITION_NUMBER")
+function shrink_adjacent_partition(){
+    local device_type=
```

Comment:
I think the `=` sign should be removed here, right?

Reply:
Shellcheck didn't complain, but I can remove it.